exchanges/stream/websocket_types.go

```diff
@@ -0,0 +1,108 @@
+package stream
+
+import (
+	"sync"
+	"time"
+
+	"github.com/thrasher-corp/gocryptotrader/exchanges/protocol"
+	"github.com/thrasher-corp/gocryptotrader/exchanges/stream/buffer"
+)
+
+// Websocket functionality list and state consts
+const (
+	// WebsocketNotEnabled alerts of a disabled websocket
+	WebsocketNotEnabled      = "exchange_websocket_not_enabled"
+	manageSubscriptionsDelay = 5 * time.Second
```

Comment:
unused at the moment? to be used in future or is it old code? ignore if this is to be used later :D